src/components/Questions.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Answers from './Answers';
+
+const Questions = (props) => {
+  const quiz = props;
+  if (quiz.questions) {
+  return (
+    <div>
+      {quiz.questions.map(question => { return (
+        <div key={question.id}>
+          <h3>{question.title}</h3>
+          <Answers answers={quiz.questions} />
```

Comment:
Let's loop through the answers first and then render a single `Answer` component for each. So this would look something more like:

```javascript
question.answers.map(answer, index => {
    <Answer title={answer.title} score={answer.score} index={answer.index} />
});
```